tests/linalg_test.py

```diff
@@ -2357,5 +2358,59 @@ def testPascal(self, n, kind):
     self._CompileAndCheck(jsp_fun, args_maker)
 
 
+class TestSolveSylvester(jtu.JaxTestCase):
```

Comment:
No need to make a new test class for this single function; instead let's add new test cases to the existing `ScipyLinagTest`.